GUI/MainWindow.xaml.cs

```diff
@@ -454,14 +454,21 @@ private void AddAFile(string draggedFilePath)
 
                     goto case ".mzml";
 
+                case ".mgf":
```

Comment:
maybe something like:
".mgf files lack MS1 spectra, which are needed for quantification and searching for coisolated peptides. All other features of MetaMorpheus will function."